Malayalam/sasi/pling.md

```diff
@@ -1 +1,3 @@
-പ്ലിംഗ്  എന്നാൽ ചമ്മുക , അല്ലാതെ  എന്ത് . ചുരുക്കി പറഞ്ഞാൽ ശശി എന്ന് നാമമുള്ളവൻ എന്നും പ്ലിങ്ങും 
+﻿പ്ലിംഗ്  എന്നാൽ ചമ്മുക , അല്ലാതെ  എന്ത് . ചുരുക്കി പറഞ്ഞാൽ ശശി എന്ന് നാമമുള്ളവൻ എന്നും പ്ലിങ്ങും 
+ശശി ജനിച്ചത് [അടൂർ] (../places/adoor.md) . 
+ശശിയുടെ അച്ഛൻ [ബാങ്കിൽ മാനേജർ] (../profession/banakmanager.md) ആയിരുന്നു.
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.